geopandas/tests/test_sindex.py

```diff
@@ -22,8 +22,20 @@ class TestNoSindex:
     @pytest.mark.skipif(sindex.has_sindex(), reason="Spatial index present, skipping")
     def test_no_sindex_installed(self):
         """Checks that an error is raised when no spatial index is present."""
+        t1 = Polygon([(0, 0), (1, 0), (1, 1)])
+        t2 = Polygon([(0, 0), (1, 1), (0, 1)])
+        s = GeoSeries([t1, t2])
+        d = GeoDataFrame({"location": [t1, t2]}, geometry="location")
```

Comment:
```suggestion
        d = GeoDataFrame({"geometry": [t1, t2]}, geometry="geometry")
```
(using standard convention is less distracting)

Reply:
implemented